ZAPD/ZAnimation.cpp

```diff
@@ -26,14 +25,6 @@ void ZAnimation::ParseRawData()
 
 void ZAnimation::Save(const fs::path& outFolder)
 {
-	if (Globals::Instance->testMode)
-	{
-		HLAnimationIntermediette* anim = HLAnimationIntermediette::FromZAnimation(this);
-		std::string xml = anim->OutputXML();
-		File::WriteAllText(outFolder / (name + ".anmi"), xml);
-
-		delete anim;
-	}
 }
```

Comment:
Wouldn't be a better idea to simply remove the whole function instead of leave it empty?